pkg/agent/openflow/client.go

```diff
@@ -626,6 +626,12 @@ func (c *client) initialize() error {
 			return fmt.Errorf("failed to setup policy only flows: %w", err)
 		}
 	}
+	if err := c.ovsctlClient.AddMeterEntry(binding.NPPacketInMeterId, binding.NPPacketInRate); err != nil {
+		return fmt.Errorf("failed to install OpenFlow meter entry (meterId:%d, rate:%d) for NetworkPolicy packet-in rate limiting: %v", binding.NPPacketInMeterId, binding.NPPacketInRate, err)
+	}
+	if err := c.ovsctlClient.AddMeterEntry(binding.TFPacketInMeterID, binding.TFPacketInRate); err != nil {
```

Comment:
I don't see this meter being used anywhere?

Reply:
Changed.